### PR TITLE
Fix PEtab import (Priors)

### PR DESCRIPTION
**Pull request by jvanhoefer**
bugfix + compliance to current PEtab version in prior import

### CODE REVIEW

File: pypesto/petab/importer.py [hunk 1]
```diff
@@ -283,10 +283,8 @@ def create_prior(self) -> NegLogParameterPriors:
                 prior_type_entry = self.petab_problem.\
                     parameter_df.loc[x_id, petab.OBJECTIVE_PRIOR_TYPE]
 
-                # TODO: Change the hardcoded "uninformative" to
-                #  petab.UNINFORMATIVE, if corresponding PEtab PR is merged
-                if not (np.isnan(prior_type_entry)
-                        or prior_type_entry == 'uninformative'):
+                if (not isinstance(prior_type_entry, str)
```
[the remark this answers is not in the document]
Comment (dilpath) [reply pypesto/petab/importer.py:286]: `prior_type_entry` is the string in the `initializationPriorType` or `objectivePriorType` in the [PEtab parameters table](https://petab.readthedocs.io/en/stable/documentation_data_format.html#parameter-table) and `np.isnan()` fails on strings.

[the remark this answers is not in the document]
Comment (yannikschaelte) [reply pypesto/petab/importer.py:286]: yes, agreed on that. but you want to go into the if clause, if prior_type_entry is a string _and_ is not parameter_scale_uniform_ right?

[the remark this answers is not in the document]
Comment (dilpath) [reply pypesto/petab/importer.py:286]: Yes agreed, had a different issue that was just encountered in mind. Just tested `isinstance string and not uniform prior` and it seems to work.

[the remark this answers is not in the document]
Comment (jvanhoefer) [reply pypesto/petab/importer.py]: Thank you, should be fine now :)

**Review by FFroehlich — APPROVED** [review]

**Review by yannikschaelte — APPROVED** [review]

**Comment by jvanhoefer** [comment]
Third try for the sampling tests worked... If this stays consistent throughout other PRs, we might want to adjust the rejection thresholds in those tests :)

**Comment by yannikschaelte** [comment]
> Third try for the sampling tests worked... If this stays consistent throughout other PRs, we might want to adjust the rejection thresholds in those tests :)

Definitely. If you notice some of the tests failing too often, no problem to reasonably adjust thresholds.

**Pull request merged**